pkg/flagutil/flagutil_test.go

```diff
@@ -17,15 +17,39 @@
 package flagutil
 
 import (
+	"bufio"
+	"errors"
 	"fmt"
+	"os"
+	"reflect"
+	"runtime"
 	"sort"
+	"strings"
 	"testing"
 
 	"gotest.tools/v3/assert"
 )
 
-func TestReplaceOrAppendEnvValues(t *testing.T) {
+// tmpFileWithContent will create a temp file with given content.
+func tmpFileWithContent(t *testing.T, content string) string {
```

Comment:
Why not use `t.TempDir`

Reply:
It looks like `t.TempDir` returns a path. Do you mean I should use this path for the going test?

Reply:
yes

Reply:
Make sense. Done.